pkg/rancher-desktop/config/help.ts

```diff
@@ -0,0 +1,8 @@
+class HelpImpl {
+  url(key: string | undefined): string {
+    // ToDo , to map urls with key
```

Comment:
Tiny quibble, this should be `// TODO: to map URLs with key` 
to be consistent with all the other `TODO`s.

Reply:
fixed!